packages/site/index.html

```diff
@@ -4,7 +4,8 @@
 <head>
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Atlantis@next</title>
+  <title>Atlantis</title>
+  <link rel="icon" type="image/png" href="./public/atlantis_favicon.svg" />
```

Comment:
nit: I'm not sure how much it matters, since the favicon seems to be loading across browsers, but at the bottom of this [wiki](https://en.wikipedia.org/wiki/Favicon) it suggests to use `type="image/svg+xml"` since we're using an `svg` and not a `png`.

Reply:
Nice catch! updated